### ISSUE TITLE
SOD - Warlock Tank is not an Offtank -> Plates are not blue but red when he has aggro

### ISSUE THREAD

**Issue by kolroser** [Backupiseasy/ThreatPlates#507]
Hey,

as the title describes, in SOD warlocks can be tanks.
But when a warlock tank has aggro, the threatplates look red instead of blue, to the other tank.
The reason is probably because a warlock was never a tank and the addon does not recognize the warlock as an off-tank.
Can you either fix this or tell me how I can? :)

**Comment by Backupiseasy** [comment]
Ah, it really was a long time ago I worked on that, I just realized. Basically, a tanking unit is only shown as off-tank in a raid/dungeon situation when the tanking unit has the role tank. Otherwise, it will be shown red (pulling aggro).

In Classic, it's even more complicated as role do not really exists as far as I am aware. So, I use the party assignment. If the tanking unit is assigned as tank, it should show as off-tank, otherwise not. There really is not any better way to implement this, at least not as far as I can think off. 

Does that match to your situation?
